### PR TITLE
Var menu has a dropdown for granting martial arts.

### PR DESCRIPTION
**Pull request by tralezab**
I was making a ninjutsu martial art, and I got really pissed that I have to make specific snowflake items whenever I want to test a martial art, so I coded a quick fix for that in the dropdown. Then, I realized that admins have to find these same dumb snowflake art granters and so I decided to quickly make a new branch and pr this.

:cl:
admin: admins, you can now find martial art granting buttons in your VV dropdown.
/:cl:

Why: Way better way of doing martial art TC trades, hopefully saves some time for our admins. Maybe helpful in setting up events.

### CODE REVIEW

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -998,6 +998,33 @@
 			manipulate_organs(C)
 			href_list["datumrefresh"] = href_list["editorgans"]
 
+		else if(href_list["givemartialart"])
+			if(!check_rights(NONE))
+				return
+
+			var/mob/living/carbon/C = locate(href_list["givemartialart"]) in GLOB.mob_list
```
Comment (Cyberboss) [inline code/datums/datumvars.dm]: GLOB.carbon_list iirc

**Comment by tralezab** [comment]
https://i.imgur.com/wesYVY5.png demonstration, if you needed one.

File: code/datums/datumvars.dm (hunk 1 above)
Comment (tralezab) [reply code/datums/datumvars.dm]: I thought this would mess up the carbon check, but it works fine. Good find.

**Comment by Bawhoppen** [comment]
Jeez. When did we get so many types of martial arts?

**Comment by ExcessiveUseOfCobblestone** [comment]
> mushroom punch

f

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**